exercises/string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,20 @@
+/* exported numVowels */
+/* input - string --- output - number
+-create counter for number
+-check each character to see if it is a vowel
+-if character is a vowel, increase counter by 1
+-return counter */
+
+function numVowels(string) {
+  var counter = 0;
+  for (var i = 0; i < string.length; i++) {
+    if ((string[i].toUpperCase() === 'A') ||
+      (string[i].toUpperCase() === 'E') ||
+      (string[i].toUpperCase() === 'I') ||
+      (string[i].toUpperCase() === 'O') ||
+      (string[i].toUpperCase() === 'U')) {
```

Comment:
Rather than calling upperCase over and over again here, you should uppercase the entire string at the beginning of the function, and then you can just compare the characters without needing to call a method for each condition.